### PR TITLE
Insertion of error messages at agreed points

### PR DESCRIPTION
**Pull request by breycarr**
added error message when the user passwords do not match
Co-authored-by: Ben Reynolds-Carr <bjr.kitz@gmail.com>
Co-authored-by: hughbric <mhughes36@live.co.uk>

### CODE REVIEW

File: views/index.erb [hunk 1]
```diff
@@ -12,4 +12,11 @@
     <input type='password' name='password_confirmation' placeholder='Confirm Password'>
     <input type='submit' value='Sign up'>
   </form>
+
+  <% if flash[:nomatch] %>
+    <div id='flash' class='nomatch'>
+      <a class="close" data-dismiss="alert"><%= flash[:nomatch] %></a>
+    </div>
+  <% end %>
+
```
Comment (olliebrownlow) [inline views/index.erb]: I'm sure we can simply use:   <%= flash[:nomatch] %> in the view.

File: views/login.erb [hunk 2]
```diff
@@ -11,4 +11,10 @@
       <input type='password' name='password' placeholder='Password'>
       <input type='submit' value='Log in'>
     </form>
+
+    <% if flash[:nomatch] %>
+      <div id='flash' class='nomatch'>
+        <a class="close" data-dismiss="alert"><%= flash[:nomatch] %></a>
+      </div>
+    <% end %>
```
Comment (olliebrownlow) [inline views/login.erb]: Likewise here: <%= flash[:nomatch] %>

File: views/index.erb (hunk 1 above)
Comment (breycarr) [reply views/index.erb]: That's good to know, I've updated the relevant parts of code, and confirmed that it still passes rspec and performs as expected in rackup.

File: views/login.erb (hunk 2 above)
Comment (breycarr) [reply views/login.erb]: As above, refactored in commit fb0f413

**Pull request merged**